src/components/cards/TripCard/TripCard.styles.js

```diff
@@ -0,0 +1,14 @@
+export const root =
+  "su-group su-relative su-overflow-hidden su-bg-black-true su-break-words su-border-su-border-black";
+export const content = "su-rs-px-1 su-rs-pb-3 su-text-white";
+export const image =
+  "su-w-full su-h-full su-transition-transform su-transform-gpu group-hover:su-scale-[1.03] group-focus-within:su-scale-[1.03]";
+export const date =
+  "su-relative su-mt-[-5.6rem] lg:su-mt-[-6.3rem] su-mb-26 su-z-10";
+export const link =
+  "su-stretched-link su-z-20 su-rs-mt-0 su-mb-08em su-no-underline hocus:su-underline su-underline-offset !su-underline-thick !su-underline-digital-red-xlight";
```

Comment:
```suggestion
  "su-stretched-link su-z-20 su-rs-mt-0 su-mb-08em su-type-3 su-no-underline hocus:su-underline su-underline-offset !su-underline-thick !su-underline-digital-red-xlight";
```
Give this a try, in combination with the below su-type-0 in heading